test/mason/masonInit/masonInitPathTest.chpl

```diff
@@ -5,9 +5,10 @@ use MasonInit;
 use MasonUtils;
 
 proc main(){
-  const args = ['init','testSrc'];
+  const args: [0..2] string;
```

Comment:
Ah, so the issue was that `masonInit` expects args to be 0-based. Would it be difficult to instead make `masonInit` support any indices? You can get the low value with `args.domain.low` for example.

Reply:
@ben-albrecht I applied `args.domain.low` to the code. Now, it works fine for `mason new projectName` or `mason init projectName` but doesn't work for `mason init project-Name --name project` or for mason new. However, the typed command on terminal works perfectly. I think we keep the `args` declarations for tests involving the flag.

Reply:
In `mason new project-Name --name project` , it creates a module by name of new.

Reply:
>  but doesn't work for `mason init project-Name --name project` or for `mason new`.

That seems odd. Is that just a bug in the arg parser then?